src/App.js

```diff
@@ -1,71 +1,20 @@
-import React, { Component } from 'react'
-
-// Config for API (fetch products..)
-import config from './config.js'
-
-// get token for authorization our data source
-const getToken = () =>
-  fetch(`${config.apiUrl}/oauth/token`, {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/json',
-    },
-    body: JSON.stringify({
-      grant_type: 'client_credentials',
-      client_id: config.clientId,
-      scope: config.scope,
-    }),
-  }).then(res => res.json())
-
-// get all products list (skus) from API
-const getSkus = access_token =>
-  fetch(`${config.apiUrl}/api/skus`, {
-    headers: {
-      'Content-Type': 'application/json',
-      Authorization: `Bearer ${access_token}`,
-    },
-  }).then(res => res.json())
-
-class App extends Component {
-  state = {
-    isLoading: true, // stop confusing users, what happening until fetch data
-    products: {},
-  }
-
-  async componentDidMount() {
-    const { access_token } = await getToken()
-    const products = await getSkus(access_token)
-
-    this.setState({ products, isLoading: false })
-  }
-
-  render() {
-    const {
-      isLoading,
-      products: { data },
-    } = this.state
-
-    return (
-      <div>
-        <h1>E-Commerce app</h1>
-        {isLoading && 'loading...'}
-        {data && (
-          <ul>
-            {data.map(item => (
-              <li key={item.id}>
-                <h2>{item.attributes.name}</h2>
-                <img
-                  src={item.attributes.image_url}
-                  height="60"
-                  alt={item.attributes.description}
-                />
-              </li>
-            ))}
-          </ul>
-        )}
-      </div>
-    )
-  }
-}
+import React from 'react'
+import { Link } from 'react-router-dom'
+import Urls from './constants/urls'
+
+const App = () => (
```

Comment:
This file changed its purpose since beginning and it deserves to be renamed to something like Header and it also needs to be included on each page, not in index.js. Reason for this is that <Link> component is used outside of <Router>

Reply:
this <Header> component can be placed in Components folder